components/abtesting/optimizelyXExperiment/README.md

```diff
@@ -41,6 +41,21 @@ Component will obtain OptimizelyX's decision for given `experimentId` and displa
 
 If no variation is provided or OptimizelyX API is unavailable, `defaultVariation` will be displayed.
 
+### Optional: Empty variation
+
+If you need to put an empty variation, **do not use `<Fragment>` component as you will get a lot of warnings message in the console**. Instead, use the `<EmptyVariation>` component and pass the props of the experiment.
```

Comment:
maybe instead of "to put" you could use "to use"

I think it should be "a lot of warning messages" and "component and pass by the experiment's props"

Reply:
Thanks!

Reply:
Thanks! Fixed :)